dodola/cli.py

```diff
@@ -51,6 +51,27 @@ def dodola_cli(debug):
     logging.basicConfig(level=loglevel)
 
 
+@dodola_cli.command(help="Clean up and standardize GCM")
+@click.argument("x", required=True)
+@click.argument("out", required=True)
+@click.option(
+    "--drop_leapdays/--no-drop_leapdays",
+    default=True,
+    help="Whether to remove leap days",
+)
+def cleancmip6(x, out, drop_leapdays):
+    """Clean and standardize CMIP6 GCM to 'out'. If drop-leapdays option is set, remove leap days"""
+    services.clean_cmip6(x, out, drop_leapdays)
```

Comment:
Ah, this doesn't grab authentication configs for storage. We need to authenticate storage and pass it to the service here.

So, we instantiate storage (because right now, in cli.py, we have access to CLI and user-input configs) and pass the authenticated storage to the service like `storage=_authenticate_storage()`. Need to pass something to that required `storage` arg.

Reply:
Ahh I definitely did not mean to leave that out. Thanks for catching it.